### PR TITLE
Use new OrangePi5 images and add OrangePi5 Pro

### PR DESCRIPTION
**Pull request by crschardt**
Base builds on the new OrangePi images, which includes support for the OrangePi5 Pro.

### CODE REVIEW

**Comment by crschardt** [comment]
Tested the image created by this and it works fine on my OrangePi5. Please review and I'll merge.

**Review by mcm001 — APPROVED** [review]
Are you a deceptively simple PR? Because you sure LGTM 👍

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -389,7 +389,7 @@ jobs:
   build-image:
     needs: [build-package]
 
-    if: ${{ github.event_name != 'pull_request' }}
+    # if: ${{ github.event_name != 'pull_request' }}
```
Comment (Juniormunk) [inline .github/workflows/build.yml:392]: uncomment before merging?

**Review by Juniormunk — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (crschardt) [reply .github/workflows/build.yml]: Good catch. I thought I had pushed that fix at the same time as I had marked this for review. Fixed now.

**Review by Juniormunk — APPROVED** [review]

**Pull request merged**